src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -231,7 +234,7 @@ func pollWithHttpClient(currentCfg yamlconf.Config, client *http.Client) (mutate
 	// We don't pass an auth token here, as the http client is actually hitting the localhost
```

Comment:
This line of comment seems odd as auth token is not something `config` package should care.